updateCli/updateCli.d/charts/falco.tpl

```diff
@@ -1,9 +1,11 @@
-source:
-  kind: helmChart
-  spec:
-    url: https://falcosecurity.github.io/charts
-    name: falco
-
+title: Bump falco helm chart version
+pipelineID bumpfalcohelmchart
```

Comment:
```suggestion
pipelineID: bumpfalcohelmchart
```

Reply:
Thanks, I have to double check why I didn't catch that :/